OpenRA.Mods.Common/AI/AIDamageReactionModule.cs

```diff
@@ -0,0 +1,121 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Pathfinder;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.AI
+{
+	[Desc("Manages how AI reacts to its actors receiving damage.")]
+	public class AIDamageReactionModuleInfo : IAIModuleInfo
+	{
+		[Desc("Name for identification purposes.")]
+		public readonly string Name = "default-damagereaction-module";
```

Comment:
Why is `default` in this name?

Reply:
If we're going to have modules added by default shouldn't the harvester one also be a default?

Reply:
That was a relic from earlier code revisions.